herd/machModelChecker.ml

```diff
@@ -306,6 +306,22 @@ module Make
                  "F", E.is_barrier;
                  "DATA", is_data_port;
                  "NDATA", (fun e -> not (is_data_port e));])) in
+      let evts_map = E.EventSet.fold
+                (fun e evts_map ->
+                  let attrs = S.A.V.memattr_as_list (E.Act.memattr_of e.E.action) in
+                  List.fold_right
+                    (fun attr evts_map ->
+                      let evts_w_attr =
+                        match StringMap.find_opt attr evts_map with
```

Comment:
There is a `StringMap.safe_find` _default_  _key_ _map_ function that would enable writing terser code (see `lib/MyMap.ml`).

Reply:
The code has changed slightly but the new version uses `StringMap.safe_find`